### PR TITLE
Ahmad/DTRA-1119/Add Arabic Language

### PR DESCRIPTION
**Pull request by ahmadtaimoor-deriv**
Added arabic language config

### CODE REVIEW

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

File: sass/styles/chart.scss [hunk 1]
```diff
@@ -77,6 +77,12 @@ body {
     }
 }
 
+/*! rtl:ignore */
+.sharing .ciq-no-share {
```
Comment (maryia-deriv) [inline sass/styles/chart.scss:81]: @ahmadtaimoor-deriv where are these classNames used? cannot find usages.
And if we are using them, where is it overriden ? why are we using `!important`'s ? :)

**Review by maryia-deriv — CHANGES_REQUESTED** [review]

**Review by balakrishna-deriv — APPROVED** [review]

File: sass/styles/chart.scss (hunk 1 above)
Comment (ahmadtaimoor-deriv) [reply sass/styles/chart.scss]: Removed now , thanks

**Review by kate-deriv — APPROVED** [review]

**Review by suisin-deriv — APPROVED** [review]

**Review by maryia-deriv — APPROVED** [review]

**Pull request merged**